app/controllers/invoices_controller.rb

```diff
@@ -30,7 +30,7 @@ def create
     if @invoice.save
       if params[:email_send] == 'true'
         send_invoice_to_client
-        @invoice.update_columns(status: "sent")
+        @invoice.update_columns(status: Invoice.statuses[:sent])
```

Comment:
use `@invoice.sent!` syntax for updating enum column.

Reference https://hackhands.com/ruby-on-enums-queries-and-rails-4-1/